_ont/ont-huawei-ma5671a-root-web.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Root Procedure for Huawei MA5671A (V3)
+has_children: false
+parent: Huawei MA5671A
+layout: default
+---
+
+# Root the stick
+
+Connect the SFP adapter to the TTL adapter according to the following diagram:
+
+| USB TTL (UART) Adapter | wire colour in picture | SFP 20pins Molex connector |
+| ---------------------- | ---------------------- | -------------------------- |
+| 3.3V                   | red                    | 3.3 (pin #15 or #16)       |
```

Comment:
```suggestion
| 3.3V                   | red                    | 3.3 (pin #15 and #16)       |
```

Both pins should be connected to 3.3V according to the SFP pinout, a proper SFP to PIN adapter should short them together.